lib/cl/ensure-externals.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -d "externals/" ] ; then
```

Comment:
What happens if externals are partially populated due to a crash?  Can one run the same quicklisp command idempotently?  If so, I would remove this if statement and just always run quicklisp to make sure externals are correct.  You could perhaps make the curl command smarter to only update the file if it has changed - I suspect curl has an option for that, but I don't know...

Reply:
it is possible. Fixed in the pushed commit (some extra eval forms are explained in the commit message.